openc3-init/plugins/packages/openc3-tool-common/src/services/openc3-api.js

```diff
@@ -119,8 +119,8 @@ export class OpenC3Api {
     return this.exec('map_target_to_interface', [target_name, interface_name])
   }
 
-  connect_interface(interface_name) {
-    return this.exec('connect_interface', [interface_name])
+  connect_interface(interface_name, interface_params) {
```

Comment:
interface_params should default to nil or an empty array

Reply:
Null? this is the javascript side. This is the part I wasn't sure about. I think probably an empty array.